### PR TITLE
Create 2022-08-08-protocol.md for chickenshock meeting

### PR DESCRIPTION
**Pull request by marlau97**

### CODE REVIEW

File: protocols/minigames/chickenshock/2022-08-08-protocol.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+What do we still want to add to chickenshock / what would be nice to have, what is left to do this sprint?
+
+- present: Martin, Levi
+- absent excused: -
+- absent unexcused: -
+
+## What do we want to add?
+
+- Pause menu with options menu and quit button.
+- If the player pauses he loses 1 point per question to prevent abuse of pausing
```
Comment (delvh) [inline protocols/minigames/chickenshock/2022-08-08-protocol.md:12]: Alternatively, the question could be skipped.
Or the lecturer can set delays per configuration to show a question only for the set amount of time.

File: protocols/minigames/chickenshock/2022-08-08-protocol.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+- Should the pause menu stop the timer? Not 100% decided, but probably no
+- Option to change mouse sensitivity
```
Comment (delvh) [inline protocols/minigames/chickenshock/2022-08-08-protocol.md:14]: Not your job?
That's the job of the OS?

File: protocols/minigames/chickenshock/2022-08-08-protocol.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+- Change the overlay of the game info so it can hold more text and is better scaleable with screen size
+- A maximum magazine size of 2 to fit the shotgun theme, each question starts with 2 shots and if you need more you'll have to reload
+- A small animation for realoading the weapon
+- Game time is not static and can be configured by the lecturer, but there is a default time in case it is not set by the lecturer
+- End screen can show what you answered to the question and indicates if it was right/shows
```
Comment (robojones) [inline protocols/minigames/chickenshock/2022-08-08-protocol.md]:
```suggestion
- End screen can show what you answered to the question and indicates if it was right / shows
```

File: protocols/minigames/chickenshock/2022-08-08-protocol.md (hunk 2 above)
Comment (marlau97) [reply protocols/minigames/chickenshock/2022-08-08-protocol.md:14]: What? If it was then every single shooter on the market wouldn't need a mouse sens option

File: protocols/minigames/chickenshock/2022-08-08-protocol.md (hunk 3 above)
Comment (marlau97) [reply protocols/minigames/chickenshock/2022-08-08-protocol.md]: Changed to @delvh suggestion

**Review by robojones — APPROVED** [review]

File: protocols/minigames/chickenshock/2022-08-08-protocol.md (hunk 2 above)
Comment (delvh) [reply protocols/minigames/chickenshock/2022-08-08-protocol.md:14]: Okay, if you want to add it, then go for it.
But I would think that the user should have configured his system to his liking...

**Review by delvh — APPROVED** [review]

File: protocols/minigames/chickenshock/2022-08-08-protocol.md (hunk 1 above)
Comment (marlau97) [reply protocols/minigames/chickenshock/2022-08-08-protocol.md:12]: Not a bad idea, we will keep it in mind

File: protocols/minigames/chickenshock/2022-08-08-protocol.md (hunk 2 above)
Comment (marlau97) [reply protocols/minigames/chickenshock/2022-08-08-protocol.md:14]: I think there's a misunderstanding here, we don't project the crosshair on you cursor, instead we check for the cursor movement and use that to move the camera. This means we have a (currently static) multiplier to you mouse movement which we want to make variable so you have to move the mouse more or less to move the camera more or less. If that makes sense

**Pull request merged**